File: pset6/mario/mario.py
from cs50 import get_int

def main():
    height = get_positive_int()
    
    for i in range(1,height+1):
        for j in reversed(range(height)):
            if j >= i:
                print(" ", end="")
            else:
                print("#", end="")
        
        print("  ", end="")
        
        for k in range(i):
            print("#", end="")
        
        print()
    
def get_positive_int():
    while True:
        n = get_int("Height: ")
        if n > 0 and n < 9:
            break
    return n

main()

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void) {
    int height = get_int("Height: ");
    while(height < 1 || height > 8) {
        height = get_int("Height: ");
    }

    for (int i = 1; i <= height; i++) {
        for(int j = height; j > 0; j--) {
            if(j > i) {
                printf(" ");
            } else {
                printf("#");
            }
        }
        
        printf("  ");
        
        for(int k = 0; k <i ; k++) {
            printf("#");
        }
        printf("\n");
    }
}